flask_stormpath/views.py

```diff
@@ -36,66 +36,43 @@ def register():
     template that is used to render this page can all be controlled via
     Flask-Stormpath settings.
     """
-    form = RegistrationForm()
+    form = RegistrationForm(config=current_app.config)
```

Comment:
Seems like a waste to pass the entire config here - not sure if this can be optimised somehow.